src/toggle.scss

```diff
@@ -11,35 +11,114 @@ $block: #{$fd-namespace}-toggle;
 
 .#{$block} {
 
-  $fd-toggle-sizes: ("default": fd-space(13) fd-space(5.5) 20px -5px -5px,
-  "xs": fd-space(9) fd-space(5) 14px -4.5px 15px,
-  "s": fd-space(10) fd-space(6) 17px -5px 15px,
-  "compact": fd-space(10) fd-space(6.5) 16px -6px 13px,
-  "l": fd-space(14) fd-space(8) 22px -6px 23px,
-) !default;
-  $_defaults: map-get($fd-toggle-sizes, "default");
-  $fd-toggle-width: nth($_defaults, 1);
-  $fd-toggle-height: nth($_defaults, 2);
-  $fd-toggle-background-color: fd-color("neutral", 4) !default;
-  $fd-toggle-on-background-color--disabled: #c5dcf2;
-  $fd-toggle-switch-width: nth($_defaults, 2);
-  $fd-toggle-switch-background-color--disabled: hsla(240, 2%, 93%, 1) !default;
-  $fd-toggle-switch-border-color--disabled: hsla(212, 8%, 57%, 0.4) !default;
-  $fd-toggle-switch-on-background-color--disabled: #9ebbd8 !default;
-  $fd-toggle-switch-hover-background-color: #ebf5fe;
+  $fd-toggle-active-border-color: var(--sapButton_Selected_BorderColor);
+  $fd-toggle-active-background-color: var(--sapButton_Selected_Background);
+  $fd-toggle-inactive-border-color: var(--sapContent_ForegroundBorderColor);
+  $fd-toggle-inactive-switch-background: var(--sapButton_Background);
+
+  $fd-toggle-hover-border-color: var(--sapButton_Hover_BorderColor);
+  $fd-toggle-hover-selected-border-color: var(--sapButton_Selected_Hover_BorderColor);
+  $fd-toggle-hover-background: var(--sapButton_Hover_Background);
+  $fd-toggle-hover-selected-background: var(--sapButton_Selected_Hover_Background);
+  $fd-toggle-success-border-color: var(--sapSuccessBorderColor);
+  $fd-toggle-error-border-color: var(--sapErrorBorderColor);
+  $fd-toggle-border-width: var(--sapButton_BorderWidth);
+
+  $fd-toggle-label-width: 1.75rem;
+
+  $fd-toggle-transition: all 0.1s;
+
+  $fd-toggle-container-padding-y: 0.375rem;
+  $fd-toggle-size-x: 3.75rem;
+  $fd-toggle-size-y: 1.25rem;
+  $fd-toggle-switch-size: 1.875rem;
```

Comment:
Isn't it 2rem?

Reply:
It is, but actually it's counted with (2* 0.0625rem) border, so it gives 2rem.